src/Drive.php

```diff
@@ -577,6 +580,70 @@ public function emptyTrashbin(): bool
         return true;
     }
 
+    /**
+     * Invite a user or group to the drive.
+     * Every recipient will result in an own ShareCreated object in the returned array.
```

Comment:
This method returns a single `ShareCreated` object.
So I don't think that this comment is correct - the method does not return an array.